Formula/b/bozohttpd.rb

```diff
@@ -0,0 +1,44 @@
+class Bozohttpd < Formula
+  desc "Small and secure http version 1.1 server"
+  homepage "http://www.eterna.com.au/bozohttpd/"
+  url "http://www.eterna.com.au/bozohttpd/bozohttpd-20220517.tar.bz2"
+  sha256 "9bfd0942a0876e5529b0d962ddbcf50473bcf84cf5e4447043e4a0f4ea65597a"
+  license "BSD-2-Clause"
+
+  depends_on "pkg-config" => :build
+  depends_on "rtmpdump" => [:build, :test]
+  depends_on "lua"
+  depends_on "openssl@3"
+
+  uses_from_macos "curl" => [:build, :test]
```

Comment:
If we use only `curl` binary (and don't link with libcurl), we don't need to specify it here (`curl` is a dependency of brew itself, so it always exists), unless we need the latest version of `curl` because of some of its new features are being used.

Regarding build-only dependency, does it download anything in a buildtime?

Reply:
Except downloading the source code itself, nothing else needs to be downloaded during build.

Reply:
Removed build dependency `curl` and `rtmpdump`.